src/main/java/com/shaft/dsl/webElements/Button.java

```diff
@@ -0,0 +1,21 @@
+package com.shaft.dsl.webElements;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Button extends Label {
+
+    public Button(WebDriver driver, By locator) {
+        super(driver, locator);
+    }
+    public void click() {
+        elementActions.click(locator);
+    }
+    public String getText() {
+        return elementActions.getText(locator);
+    }
+    public Boolean getStatus(){
```

Comment:
I think this method is commonly knows as isEnabled()
Also I think this returns true if the element is disabled which is actually the opposite of what it should return...
We need some test methods to cover all the new methods here